my-app/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import Display from "./Components/Display";
+import Dashboard from "./Components/Dashboard";
+
+import "./App.css";
+
+const App = () => {
+  const [count, setCount] = useState(0);
+  const [ball, setBall] = useState(0);
+  const [strike, setStrike] = useState(0);
+  const [hit, setHit] = useState(0);
+  const [foul, setFoul] = useState(0);
+
```

Comment:
another way of setting state would be something like this: 
`const [stats, setStats] = useState({
    count: 0,
    ball: 0,
    strike: 0,
    hit: 0,
    foul: 0
})`